js/build.js

```diff
@@ -67,44 +65,29 @@ Fliplet.Widget.instance({
 
       function evaluate(condition, expression) {
         try {
-          eval(expression);
-
           if (eval(expression)) {
             return condition['visibility'];
           }
         } catch (error) {
-          // console.log('Expression can\'t be evaluated. Error: ' + error);
-
           return false;
         }
       }
 
       function setResult(expr) {
-        if (expr) {
-          return expr === 'hide' ? false : true;
-        }
-
-        return false;
+        result = expr && expr !== 'hide';
       }
 
-      function ifArrayIncludes(array, condition) {
-        if (array.includes(condition['user_value'])) {
-          return setResult(condition['visibility']);
-        }
+      function isConditionIncluded(array, condition) {
```

Comment:
`isConditionIncluded` suggests a function that returns `true` or `false` based on whether it "is" or "is not". This function is doing more than that and actually performing an action.

I don't fully understand the code here, but it could be renamed any of the following (or something else if theyr'e not suitable):

- doFooIfBar()
- doFoo({ toggle: true })
- verifyResult()
- evaluateResult()
- showContainer({ visible: true })